setup.py

```diff
@@ -197,7 +197,7 @@ def __str__(self):
           'test': TESTS_REQUIRE,
           'docs': [
               'Sphinx < 2; python_version < "3"',
-              'Sphinx >= 2; python_version >= "3"',
+              'Sphinx; python_version >= "3"',
```

Comment:
I admit I haven't dug into this yet, so I don't understand the reason we need this only on Python 3. Can't Sphinx 1 still be used on Python 2?

Reply:
The line before requires `Sphinx 1.x` for Python 2. Python 3 no longer has a version requirement to be able to pin a Sphinx 1.x version in zopetoolkit.

Reply:
I apologize, I was on the go between meetings and completely misread the diff.